CMake/aarch64-bionic-linux-gnu.toolchain.cmake

```diff
@@ -0,0 +1,46 @@
+set(GCC_COMPILER_VERSION "" CACHE STRING "GCC Compiler version")
```

Comment:
CMAKE is still indistinguishable from magic to my brain. As long as someone's tested that this is functional on aarch64 I think it's fine to merge.